### ISSUE TITLE
Term: Autosomal recessive germline de novo mutation

### ISSUE THREAD

**Issue by holtgrewe** [obophenotype/human-phenotype-ontology#7699]
**Preferred term label:**
Autosomal recessive germline de novo mutation

**Synonyms**
N/A

**Definition (free text, please give PubMed ID)**
A de novo variant that co-occurs with a rare recessive variant and causes a disease through recessive inheritance. cf. "Autosomal dominant germline de novo mutation HP:0025352"

**Parent term (use hpo.jax.org/app)**
Autosomal recessive inheritance HP:0000007

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
N/A

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-3051-1763

**Comment by holtgrewe** [comment]
Alternatively, it might be useful to have a more generic "de novo" flag for variants that occur de novo.

**Comment by pnrobinson** [comment]
@holtgrewe We have been working with the GenCC consortium to improve the inheritance hierarchy of the HPO.
These are the discussions to date: https://github.com/obophenotype/human-phenotype-ontology/issues?q=+label%3A%22mode+of+inheritance%22+
The suggestion was to make "de novo" a modifier term -- https://github.com/obophenotype/human-phenotype-ontology/issues/7678
So I think this is exactly what you are suggesting. Further comments or suggestions would be welcome. Most of the changes are in the develop branch and will become publicly available on the website at the next release, which will happen within the next 10 days.

If you agree, please close this issue, otherwise let's communicate!

**Comment by holtgrewe** [comment]
@pnrobinson I agree. Let's close this issue.
